src/app/components/select-field/index.html

```diff
@@ -119,6 +119,12 @@
     <p>
       The <sky-code>SkySelectField</sky-code> type extends the <sky-code>any</sky-code> type and supports values for data objects that you can inject into the select field component to populate the select field modal.
     </p>
+    <sky-demo-page-property
+      propertyName="description"
+      isOptional="true"
+    >
+    Specifies a description to display below each label when <sky-code>selectMode</sky-code> is set to <sky-code>multiple</sky-code>. This property accepts <sky-code>string</sky-code> values.
```

Comment:
I'd add "for an item" after "description" and change "each" to "its" ... I mean, "each" makes it sounds like this propert specifies a single description for all items, but I assume this is a per-item description not a generic one, right? ... Other than that, LGTM.